### ISSUE TITLE
release 3.3

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/PermaplanT#879]
### Tasks

- [x] run external link check (change follow-web-links in book.toml and do `mdbook build`)
- [x] fix _real_ security problems
      unfortunately `npm audit --omit=dev fix` does [too much and too little](https://overreacted.io/npm-audit-broken-by-design/),
      `cargo deny` seems to be more helpful
- [x] check/improve reformatting
- [x] check if all issues labelled `release critical` are fixed
- [x] update mergedDatasets.csv (skipped for this release)
- [x] manually test dev.permaplant.net according to protocol
- [x] build <https://build.libelektra.org/job/PermaplanT-Release/>
- [x] git tag -s vX.X.X
- [x] git push --tags
- [x] create release PR to pump versions and new section in Changelog
- [x] write announcement

### Use case

seeds

### Related Pull request

#869

**Comment by markus2330** [comment]
> [It is not possible to continue efficiently for now.](https://github.com/ElektraInitiative/PermaplanT/pull/869#issuecomment-1696070440)

Thank you for writing, I agree. I answer here as this is basically release coordination. I took the liberty to create the release issue, @4ydan you can create it by yourself the next time.

I propose following:

1. @badnames should create a PR which fixes all the [release critical bugs](https://github.com/ElektraInitiative/PermaplanT/issues?q=is%3Aissue+is%3Aopen+label%3A%22release+critical%22)
2. in the meanwhile @tschawax does the best to fix as many UI bugs as possible (in another PR)
3. if (1) and (2) is merged, @4ydan can continue e2e tests and do the manual tests for the release (and everything else described in this issue, although some of these points could already be prepared/discussed before)

**Comment by 4ydan** [comment]
I think we can release today?

**Comment by markus2330** [comment]
Yes, that would be great. Do all the e2e seed tests already work? #869

#881 would be great to have in the release, imho it only needs some testing (ideally automated e2e)

**Comment by 4ydan** [comment]
> Do all the e2e seed tests already work?

Not yet

**Comment by markus2330** [comment]
Please write news here when something works or something still fails.

**Comment by 4ydan** [comment]
I would like to perform the release friday if its okay, so I can take my time for it. I was very busy the past days with a large homework in 185.208 Programming Languages

**Comment by markus2330** [comment]
> I would like to perform the release friday if its okay, so I can take my time for it.

Yes, great. Maybe @temmey, @badnames and @tschawax can do a few more bug fixes till then.

**Comment by 4ydan** [comment]
I fixed the linting and did the small requests changes to the placeholders in #902
I will begin with the release procedure once 902 is merged.

**Comment by markus2330** [comment]
Great, thank you!

I think we can do the release.

**Comment by 4ydan** [comment]
`cargo deny` needs rust 1.70, but the project is set to 1.67.1

**Comment by 4ydan** [comment]
It seems we have manual test cases for features that are not implemented yet (heatmap, chat).

**Comment by 4ydan** [comment]
> update mergedDatasets.csv

How exactly is this to be done, so I dont make any mistake.

**Comment by markus2330** [comment]
> cargo deny needs rust 1.70, but the project is set to 1.67.1

Where is it used?

> It seems we have manual test cases for features that are not implemented yet (heatmap, chat).

Yes, they are commented out.

> See PR below this comment.

I didn't find it?

>  How exactly is this to be done, so I dont make any mistake.

For this release it is not needed, as it did not change.

**Comment by 4ydan** [comment]
> Where is it used?

It is mentioned in the release procedure as a better alternative to `npm audit --omit=dev fix`

**Comment by 4ydan** [comment]
Done from my side. Hope everything went well, let me know if there is something that I forgot or could have been done better.

**Comment by markus2330** [comment]
Great job! :heart: 

@yvonnemarkl already added some seeds.
